sympy/printing/pretty/pretty.py

```diff
@@ -79,8 +81,15 @@ def _print_atan2(self, e):
 
     def _print_Symbol(self, e, bold_name=False):
         symb = pretty_symbol(e.name, bold_name)
-        return prettyForm(symb)
+        if self._settings.get("use_unicode", True):
+            return prettyForm(symb)
+        else:
+            if has_function:
+                return prettyForm(e.name)
+            else:
+                 return symb
```

Comment:
```suggestion
                return symb
```
Indent not a multiple of 4